jq.c

```diff
@@ -277,6 +289,96 @@ jdbc_detach_jvm()
 	(*jvm)->DetachCurrentThread(jvm);
 }
 
+static void jdbc_get_jni_env(void)
+{
+	int JVMEnvStat;
+
+	ereport(DEBUG3, (errmsg("In In jdbc_get_jni_env")));
```

Comment:
It is a typo I suppose?
```suggestion
	ereport(DEBUG3, (errmsg("In jdbc_get_jni_env")));
```

Reply:
Yeah, it's a typo. Thank you for pointing this out.

Reply:
Addressed by https://github.com/scalar-labs/jdbc_fdw/pull/1/commits/ce1a3c3db82618f67e2693995558b469874f3665

Reply:
The typo still appears to be present. Could you have a look again?

Reply:
Sorry. I was mistakenly confusing what to stage with what to unstage 😢 . I fixed the commit and force-pushed. Please take a look https://github.com/scalar-labs/jdbc_fdw/pull/1/commits/399b6e69f768499b58db7c9e7f8ebeac859b4c7f

Reply:
No problem, thanks!